input/pagecontent/reference.md

```diff
@@ -957,91 +1155,112 @@ Defining extensions is similar to defining a profile, except that the parent of
+  * extension[detailed] ^short = "Extended ethnicity codes"
+  * extension[detailed].value[x] only Coding
+  * extension[detailed].valueCoding from DetailedEthnicity (required)
+  * extension[text] ^short = "Ethnicity text"
+  * extension[text].value[x] only string
+  ```
 
 * Define an extension with an explicit parent, specializing the US Core Birth Sex extension:
 
-```
-Extension:      BinaryBirthSexExtension
-Parent:         USCoreBirthSexExtension
-Id:             binary-birthsex
-Title:          "Binary Birth Sex Extension"
-Description:     "As of 2019, certain US states only allow M or F on birth certificates."
-* valueCode from BinaryBirthSex (required)
-```
+  ```
+  Extension:      BinaryBirthSexExtension
+  Parent:         USCoreBirthSexExtension
+  Id:             binary-birthsex
+  Title:          "Binary Birth Sex Extension"
+  Description:     "As of 2019, certain US states only allow M or F on birth certificates."
+  * valueCode from BinaryBirthSex (required)
+  ```
 
 #### Defining Instances
 
-Instances are defined using the keywords `Instance`, `InstanceOf`, `Title` and `Description`. The `InstanceOf` is required, and plays a role analogous to the `Parent` in profiles. The value of `InstanceOf` can be the name of a profile defined in FSH, or a canonical URL if defined externally.
+Instances are defined using the keywords `Instance`, `InstanceOf`, `Title`, `Usage` and `Description`. The `InstanceOf` is required, and plays a role analogous to the `Parent` of a profile. The value of `InstanceOf` can be the name of a profile defined in FSH, or a canonical URL (or alias) if defined externally.
 
 Instances inherit structures and values from their StructureDefinition (i.e. fixed codes, extensions). Fixed value rules are used to set additional values.
 
-**Examples:** 
-```
-Instance:   EveAnyperson
-InstanceOf: http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient
-Title:      "Eve Anyperson"
-* name.given = "Eve"
-* name.given[1] = "Steve"
-* name.family = "Anyperson"
-* birthDate = 1960-04-25
-* us-core-race.ombCategory.valueCoding = RACE#2106-3 "White"
-* us-core-ethnicity.ombCategory.valueCoding = RACE#21865 "Non Hispanic or Latino"
-```
+The `Usage` keyword specifies how the instance should be presented in the IG:
 
-```
-Instance:   DrDavidAnydoc
-InstanceOf: http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner
-Title:      "Dr. David Anydoc"
-* name[0].family = Anydoc
-* name[0].given[0] = David
-* name[0].suffix[0] = MD
-* identifier[NPI].value = 8274017284
-```
+* `Usage: Example` means the instance is intended as an illustration of a profile, and will be presented on the Examples tab for the corresponding profile.
+* `Usage: Definition` means the instance is a conformance item that is an instances of a resource such as a search parameter, operation definition, or questionnaire. These items will presented on their own IG page.
```

Comment:
I think this line might be a little too specific. Right now, if someone uses `Usage: Definition`, we look at the type of resource to determine how it is presented in the IG. It could be a conformance item, which is presented in its own page, but it could also be a vocabulary resource, logical model, operation definition, profile, or extension. If we can't categorize it into one of those, it is just presented in the the generic "resources" list. If it is helpful, I can give you a full breakdown of how we sort "Definition" instances. If it more helpful to keep this high level, I think it can be generalized to say that Definition instances will be sorted according to its `InstanceOf` type.

Reply:
It couldn't be a logical model or profile, right? The first we don't support, and the second isn't an instance. But I adjusted the language accordingly, keeping it general.

Reply:
I thought that since logical models and profiles are both StructureDefinition resources, we could technically make an instance of a StructureDefinition. We use the properties on the StructureDefinition to sort those into the supported folders for the IG generation. But keeping the language generic works for me!

Reply:
Right -- there isn't any reason a user _couldn't_ create a logical model, profile, or extension using `Instance:`; although it seems like that would be a special form of torture to me.